### ISSUE TITLE
Documentation updates

### ISSUE THREAD

**Issue by mkavulich** [NCAR/ccpp-doc#49]
The documentation has fallen out-of-date in some places, this issue will track the sections and topics that need updating and/or corrections:

 - [x] Use of optional variables in physics 
 - [x] Needs documentation of standard names
   - Introduce the concept and say that each hash of ccpp-physics has a compliant version of the standard names dictionary (information kept in top-level README.md of NCAR ccpp-physics main) 
 - [x] Compilation with stub physics capability (https://github.com/NCAR/ccpp-framework/pull/436/files)
 - [x] New debugging feature in auto-generated caps (new/changed arguments to `ccpp_prebuild.py`)
 - [x] `ccpp_error_flag` -> `ccpp_error_code` update
 - [x]  --debug flag for ccpp_prebuild.py renamed to --verbose (https://github.com/NCAR/ccpp-framework/pull/404)
 - [x] Chapter 2: when describing the phases, explain how they relate to blocking of data
 - [x] CCPP Overview: Change from _Earth System Prediction Capability_ to _Interagency Council for Advancing Meteorological Services (ICAMS; previously Earth System Prediction Capability)_
 - [x]  CCPP Overview: Update the table of supported suites/schemes to match CCPP v6 public release
 - [x]  Chapter on host side coding is **seriously** out of date, it still refers to the dynamic CCPP implementation, doesn't know about the new `timestep_init` and `timestep_finalize` phases, ...
-  [x] Replace CCPP-Framework and CCPP-Physics with CCPP Framework and CCPP Physics
-  [ ] Review the document to standardize the use of regular font, _italics_, and `this font`
-  [x] Replace examples based on the unsupported GFSv15 suite with GFSv16.
-  [x] Update "Code Management" section
-  [x] Update links to the CCPP SciDoc and SCM User's Guide to v6, once it is published
-  [x] Remove references to MRW, update description for v17_p8

Please edit this issue to add topics as you see fit.

**Comment by climbfuji** [comment]
This should not have been closed automatically, reopening.

**Comment by climbfuji** [comment]
@mkavulich Several of the items above will be addressed in my forthcoming PR https://github.com/NCAR/ccpp-doc/pull/53.

**Comment by mkavulich** [comment]
All items have been addressed aside from consistent formatting across the whole guide (though it has been done in several chapters). Since that is low priority item, I think it's safe to close this release issue once #62 is merged.

**Comment by ligiabernardet** [comment]
Hi @mkavulich , Can we close this issue since we have a new one to collect requirements for documentation (https://github.com/NCAR/ccpp-doc/issues/63)?
